config/samples/disk_formats/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+resources:
+- ../base/openstack
```

Comment:
sorry, I was wrong on L2: it should be `../backends/base/openstack`

Reply:
NP, done!